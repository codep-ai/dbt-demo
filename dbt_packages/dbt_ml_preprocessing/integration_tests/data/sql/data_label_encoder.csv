id_col,col_to_label_encode
1,paris
2,tokyo
3,sydney
4,new york
5,london
4,new york
7,new york
8,paris
9,paris
10,amsterdam