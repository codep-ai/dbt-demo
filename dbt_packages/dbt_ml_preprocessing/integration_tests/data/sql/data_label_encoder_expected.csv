id_col,col_to_label_encode,col_to_label_encode_encoded
1,paris,3
2,tokyo,5
3,sydney,4
4,new york,2
5,london,1
4,new york,2
7,new york,2
8,paris,3
9,paris,3
10,amsterdam,0
